configure.ac

```diff
@@ -234,6 +234,7 @@ OPTION_DEFAULT_DISABLE([amqp], [ENABLE_AMQP])
 
 dnl Options for contributed stores
 OPTION_DEFAULT_DISABLE([tutorial-store],[ENABLE_TUTORIAL_STORE])
+OPTION_DEFAULT_ENABLE([timescale-store],[ENABLE_TIMESCALE_STORE])
```

Comment:
@tom95858 I would like each external database plugin (rabbit, influx, sos, timescale, mysql, etc) to be disabled by default unless the checks enabling it are rather specific about version requirements.

Having our configure (or worse, build) fail on a store feature the user didn't explicitly ask for because the system has an older or newer database in the default path generates unnecessary issue traffic for us.

Reply:
@tom95858 @baallan So I made it disabled by default.

Reply:
> @morrone, @baallan, @valleydlr what do you think about this. Should it be default disable or should we use @morrone' strategy of building it automatically if the dependencies are present?

Yes, I of course believe that it should use the method of:

1.  by default check for tiemscaledb, and build if found
2. if --with/--enable option is specified, require that it be built, and error out at configure time if timescaledb not found
3. if --without/--disable option specified, don't build things that use timescaledb

I don't think there should almost ever be _store_ specific option at all. The options should be about what dependencies we want to include at build time, and then components that depend on those things should be turned on or off as a result of those selected dependencies.

In any event, this plugin does require some more build system changes. I'll comment in line.

Reply:
@tom95858 Now it works as @morrone suggests.